komodo/symlink/suggester/cli.py

```diff
@@ -94,7 +94,7 @@ def suggest_symlink_configuration(
 
     target_branch = f"{args.release}/{args.mode}"
     if "azure" in args.symlink_conf_path:
-        target_branch += "/azure"
+        target_branch = f"{args.release}/azure-{args.mode}"
```

Comment:
What about treating onprem and azure equivalently, so that `target_branch` contains either `onprem-stable` or `azure-stable`?